2020/latex/ISMIR2020template.tex

```diff
@@ -0,0 +1,251 @@
+% -----------------------------------------------
+% Template for ISMIR Papers
+% 2019 version, based on previous ISMIR templates
```

Comment:
`2020 version`, maybe?

Reply:
Absolutely! Changing now.

Reply:
Should be corrected now in release `2020v2` and properly linked on the website. Thanks for the heads up.